apps/src/lib/ui/simpleSignUp/workshop/WorkshopLinkAccountPage.tsx

```diff
@@ -3,9 +3,8 @@ import React from 'react';
 import Link from '@cdo/apps/componentLibrary/link';
 import i18n from '@cdo/locale';
 
-import WorkshopExistingAccountCard from './WorkshopExistingAccountCard';
-import WorkshopNewAccountCard from './WorkshopNewAccountCard';
-import WorkshopWelcomeBanner from './WorkshopWelcomeBanner';
+import WorkshopAccountBanner from '../../../../templates/account/AccountBanner';
```

Comment:
nit: I much prefer using the `@cdo/apps/templates/...` syntax. It's up to you if you want to change it but imo it makes these imports a lot clearer